python/paddle/fluid/layers/nn.py

```diff
@@ -232,6 +232,8 @@ def embedding(input,
               is_sparse=False,
               is_distributed=False,
               padding_idx=None,
+              allowed_beyond_boundary=False,
+              candidate_idx=-1,
```

Comment:
My feeling is that this can be replaced a few fine-grained operators. So we don't make embedding super complex.

For example

input = layers.replace_if_larger(input, min_val, replaced_val) # This can use ifelse op
embedding(input)

Reply:
I agreed with you, and I planted to add a new op to do that at first, but this backward operation is not clear.